app/views/admin/unattached_promo_registrations/index.html.slim

```diff
@@ -0,0 +1,100 @@
+.unattached-promo-registrations
+  h1 Referral Promo
+
+  hr
+
+  = form_tag admin_unattached_promo_registrations_path, id: "create-referral-codes" do
+    = label_tag "Create referral codes"
+    .
+      = number_field_tag "number_of_codes_to_create", nil, placeholder: "# codes"
+      = submit_tag "+ Referral codes", class: "btn btn-info"
+
+  = form_tag admin_promo_campaigns_path, id: "create-promo-campaign" do
+    = label_tag "Initialize a campaign"
+    .
+      = text_field_tag "campaign_name", nil, placeholder: "Campaign name"
+      = submit_tag "+ Campaign", class: "btn btn-info"
+
+  hr
+
+  = form_tag admin_unattached_promo_registrations_path, method: :get, id: "filter-by-campaign"do
+    = select_tag :filter, options_for_select(@campaigns + ["Not assigned", "All codes"], @current_campaign)
+    = submit_tag "Filter by campaign", class: "btn btn-success"
+  = form_tag admin_unattached_promo_registrations_path, method: :patch, id: "unattached-referral-code-form" do
+    table.table.table-bordered.table-striped
```

Comment:
Would you add a comment somewhere of when the stats were last synced? You should store that timestamp value in Rails.cache (aka Redis)

Reply:
And also add a comment saying that stats are adjusted every 12 hours (ideally one can read from the config file but I don't know how to do this.)

Reply:
Yes the stats are sync'd every 12 hours for all the unattached codes, but they are also sync'd when an admin generates a statement for the codes included in that statement.  

https://github.com/brave-intl/publishers/pull/1265/files#diff-a548229142426d69e713710d88d3fe0bR13

That admins are guaranteed that the stats in the statements are up to date.

I'll look into storing and displaying when codes were last sync'd for the index view.  We may want to do this for the publisher dashboard as as well since I know we get a lot of questions about that.